index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+      integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <!-- Start: Navbar-->
+      <nav>
+        <a class="logo" href="#">My Logo</a>
+        <i class="fas fa-bars"></i>
+      </nav>
+      <!-- End: Navbar-->
```

Comment:
- [ ] The project requirements specify that the container of the Toolbar section should be a header tag. However, you have used a div tag instead. [Please update the code to use a header tag instead."](https://stackoverflow.com/questions/19748487/semantic-markup-why-header-instead-of-div-id-header)[1](https://stackoverflow.com/questions/19748487/semantic-markup-why-header-instead-of-div-id-header)[2](https://discuss.codecademy.com/t/why-use-div-id-header-instead-of-header-should-we-use-html5/62596)

I hope that helps!